lib/services/ios-log-filter.ts

```diff
@@ -41,14 +41,16 @@ export class IOSLogFilter extends iOSLogFilterBase.IOSLogFilter implements Mobil
 				// This code removes unnecessary information from log messages. The output looks like:
 				// CONSOLE LOG file:///location:row:column: <actual message goes here>
 				if (pid) {
-					const searchString = "[" + pid + "]: ";
-					const pidIndex = line.indexOf(searchString);
-					if (pidIndex > 0) {
-						line = line.substring(pidIndex + searchString.length, line.length);
+					const pidIndex = line.indexOf(`[${pid}]`);
+					const ios11PidIndex = line.indexOf(` ${pid} `);
+					if (pidIndex > 0 || ios11PidIndex > 0) {
```

Comment:
again choose either `!= 1` or `> 0` for consistency [in the code](https://github.com/telerik/mobile-cli-lib/pull/1016/files#diff-6970fd71154619adb4a5ac4fc61de7fcR11)

Reply:
Could change it to `!= -1`

Reply:
I chose to use `indexOf` because its a bit more fast in my opinion(didn't really compared performance tests though).

Reply:
It is a bit faster, but I made the suggestions just for consistency sake because IMO it's more important